src/protocols/Http.ts

```diff
@@ -274,6 +274,14 @@ export default class HttpProtocol extends KuzzleAbstractProtocol {
         payload.headers.authorization = "Bearer " + value;
       } else if (key === "volatile") {
         payload.headers["x-kuzzle-volatile"] = JSON.stringify(value);
+      } else if (key === "index" || key === "collection") {
```

Comment:
According to your comment [here](https://github.com/kuzzleio/sdk-javascript/issues/713#issuecomment-1362222584), shouldn't you handle other arguments like `meta` or `headers` ?